### PR TITLE
Changes for TaxYearOverviewpage to integrate with new tax calc proxy

### PR DESCRIPTION
**Pull request by chandraAndem**

### CODE REVIEW

File: app/audit/models/AllowanceAndDeductionsResponseAuditModel.scala [hunk 1]
```diff
@@ -19,9 +19,9 @@ package audit.models
 import audit.Utilities.userAuditDetails
 import auth.MtdItUserBase
 import models.calculation.AllowancesAndDeductions
+import models.liabilitycalculation.viewModels.AllowancesAndDeductionsViewModel
```
Comment (joesheridan) [inline app/audit/models/AllowanceAndDeductionsResponseAuditModel.scala:22]: package names lower case? -> viewmodels

File: app/controllers/TaxYearOverviewController.scala [hunk 2]
```diff
@@ -19,25 +19,29 @@ package controllers
 import audit.AuditingService
 import audit.models.TaxYearOverviewResponseAuditModel
 import auth.MtdItUser
-import config.featureswitch.{CodingOut, FeatureSwitching, TxmEventsApproved}
```
Comment (joesheridan) [inline app/controllers/TaxYearOverviewController.scala:22]: are we removing txmeventsapproved feature switch?

File: app/audit/models/AllowanceAndDeductionsResponseAuditModel.scala (hunk 1 above)
Comment (chandraAndem) [reply app/audit/models/AllowanceAndDeductionsResponseAuditModel.scala]: Corrected now

File: app/controllers/TaxYearOverviewController.scala (hunk 2 above)
Comment (chandraAndem) [reply app/controllers/TaxYearOverviewController.scala:22]: Yes, these are actually added at the time of Release 6. So whereever possible we could remove it.

**Pull request merged**